scipy/special/add_newdocs.py

```diff
@@ -3035,6 +3035,52 @@ def add_newdoc(place, name, doc):
     hyp2f1(a, b, c, z)
 
     Gauss hypergeometric function 2F1(a, b; c; z).
```

Comment:
Format here should be a short definition (that part is fine) followed by a longer definition. The series definition in the notes should be moved here to be the longer definition. See e.g. https://scipy.github.io/devdocs/generated/scipy.special.eval_jacobi.html#scipy.special.eval_jacobi.